cli_tools_tests/infra/no-external-ip/main.tf

```diff
@@ -0,0 +1,65 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+variable "wrapper_executors" {
+  description = "Accounts that will execute binaries directly. The cloud build service account is included automatically."
+  default = [
+    "user:ericedens@google.com",
+    "user:zoranl@google.com",
+    "user:tzz@google.com",
```

Comment:
Having our emails in a public repo is not ideal. It's also likely we will need to store additional sensitive info (like projects names) in the future in these configs.

Can we have Terraform configurations in google3?

Reply:
Very good point. To unblock this PR, I'll remove the Terraform configurations, and we can investigate long-term solutions.

Reply:
PR description should be updated as well

Reply:
updated!